### PR TITLE
import: improve file naming

### PR DESCRIPTION
**Pull request by okJiang**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)

improve import's file naming

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [ ] master (the latest development version)
- [ ] v7.0 (TiDB 7.0 versions)
- [ ] v6.6 (TiDB 6.6 versions)
- [x] v6.5 (TiDB 6.5 versions)
- [ ] v6.4 (TiDB 6.4 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s): close https://github.com/pingcap/docs/issues/13030

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Comment by Oreoxmt** [comment]
@okJiang TiDB Cloud docs are stored in the release-6.5 branch. Please rebase this branch to release-6.5

**Review by Frank945946 — APPROVED** [review]

File: tidb-cloud/naming-conventions-for-data-import.md [hunk 1]
```diff
@@ -48,12 +48,20 @@ This section describes the naming conventions for data files. Depending on the t
...
 When you import CSV files, name the data files as follows:
 
-- `${db_name}.${table_name}[.XXXXXX].csv` ([.XXXXXX] is optional)
+`${db_name}.${table_name}[${suffix}].csv`
 
-For example:
+`${suffix}` is optional and can be one of the following formats, where *`xxx`* can be any number:
+
+- *`.xxx`*, such as `.01`
+- *`._xxx_xxx_xxx_`*, such as `._0_0_01`
```
Comment (okJiang) [inline tidb-cloud/naming-conventions-for-data-import.md:56]:
```suggestion
- *`._xxx_xxx_xxx`*, such as `._0_0_01`
```

File: tidb-cloud/naming-conventions-for-data-import.md [hunk 2]
```diff
@@ -48,18 +48,26 @@ This section describes the naming conventions for data files. Depending on the t
 
 When you import CSV files, name the data files as follows:
 
-- `${db_name}.${table_name}[.XXXXXX].csv` ([.XXXXXX] is optional)
+`${db_name}.${table_name}[${suffix}].csv`
```
Comment (qiancai) [inline tidb-cloud/naming-conventions-for-data-import.md:51]:
```suggestion
`${db_name}.${table_name}${suffix}.csv`
```

File: tidb-cloud/naming-conventions-for-data-import.md [hunk 3]
```diff
@@ -48,18 +48,26 @@ This section describes the naming conventions for data files. Depending on the t
...
 
-For example:
+`${suffix}` is optional and can be one of the following formats, where *`xxx`* can be any number:
+
+- *`.xxx`*, such as `.01`
+- *`._xxx_xxx_xxx`*, such as `._0_0_01`
+- *`_xxx_xxx_xxx`*, such as `_0_0_01`
+
+For example, all target databases and tables below are `import_db` and `test_table`:
```
Comment (qiancai) [inline tidb-cloud/naming-conventions-for-data-import.md:59]:
```suggestion
For example, the target database and table of all the following files are `import_db` and `test_table`:
```

File: tidb-cloud/naming-conventions-for-data-import.md [hunk 4]
```diff
@@ -48,18 +48,26 @@ This section describes the naming conventions for data files. Depending on the t
...
 - `import_db.test_table.01.csv`
+- `import_db.test_table._0_0_01.csv`
+- `import_db.test_table_0_0_01.csv`
 
 ### Parquet
 
 When you import Parquet files, name the data files as follows:
 
-- `${db_name}.${table_name}[.XXXXXX].parquet[.{snappy|gz|lzo}]` (`[.XXXXXXX]` and `[.{snappy|gz|lzo}]` are optional)
+- `${db_name}.${table_name}[${suffix}].parquet[.{snappy|gz|lzo}]` (`${suffix}` and `[.{snappy|gz|lzo}]` are optional)
```
Comment (qiancai) [inline tidb-cloud/naming-conventions-for-data-import.md:70]:
```suggestion
- `${db_name}.${table_name}${suffix}.parquet.{snappy|gz|lzo}` (`${suffix}` and `.{snappy|gz|lzo}` are optional)
```

File: tidb-cloud/naming-conventions-for-data-import.md [hunk 5]
```diff
@@ -82,7 +90,7 @@ For example:
 
 When you import SQL files, name the data files as follows:
 
-- `${db_name}.${table_name}[.XXXXXXX].sql` ([.XXXXXXX] is optional)
+- `${db_name}.${table_name}[${suffix}].sql` (`${suffix}` is optional)
```
Comment (qiancai) [inline tidb-cloud/naming-conventions-for-data-import.md:93]:
```suggestion
- `${db_name}.${table_name}${suffix}.sql` (`${suffix}` is optional)
```

File: tidb-cloud/naming-conventions-for-data-import.md (hunk 2 above)
Comment (hfxsd) [inline tidb-cloud/naming-conventions-for-data-import.md:51]: 需要在 ${suffix} 前面加个 . 吗？比如： 
```suggestion
`${db_name}.${table_name}[.${suffix}].csv`
```

Comment (Oreoxmt) [reply tidb-cloud/naming-conventions-for-data-import.md:51]: 不需要，suffix 中也可以不包含 .，例如 _0_0_01

File: tidb-cloud/naming-conventions-for-data-import.md (hunk 4 above)
Comment (okJiang) [reply tidb-cloud/naming-conventions-for-data-import.md:70]:
```suggestion
- `${db_name}.${table_name}${suffix}.parquet{.snappy|.gz|.lzo}` (`${suffix}` and `{.snappy|.gz|.lzo}` are optional)
```

**Review by Oreoxmt — APPROVED** [review]
LGTM

**Comment by Oreoxmt** [comment]
@qiancai PTAL again

**Review by qiancai — APPROVED** [review]

**Pull request merged**